physics/src/modules/AtmosphereBoundaryModule/FluxConfiguredAtmosphere.cpp

```diff
@@ -108,4 +108,11 @@ void FluxConfiguredAtmosphere::setData(const ModelState::DataMap& dm)
     vwind = v0;
 }
 
+void FluxConfiguredAtmosphere::update(const TimestepTime& tst)
+{
+    /* The open water heat flux is reset by the thermodynamics, so that the (slab) ocean doesn't
+     * super cool. Therefore, we have to reset it here on every update */
+    qow = qow0;
+}
+
```

Comment:
Is this required for the polynya test? It should go in the PR with the `IceGrowth` changes if it is not required for the polynya test to pass.

Reply:
The configuration can use either ``ConfiguredAtmosphere`` or ``FluxConfiguredAtmosphere``. So far, I've used ``FluxConfiguredAtmosphere`` because it's simpler, but in the literature, people have used the equivalence of ``ConfiguredAtmosphere``, so I've been switching between the two.

The bottom line is that both need these minor fixes for the configuration to work, so I think it's okay to include both in the PR, even though only one will be used in the "official" setup.